packages/qwik/src/core/use/use-core.ts

```diff
@@ -120,9 +120,6 @@ export const newInvokeContext = (
   };
 };
 
-/**
- * @alpha
- */
 export const useWaitOn = (promise: ValueOrPromise<any>): void => {
```

Comment:
@manucorporat Shouldn't this be removed?

Reply:
yes! good one